vercel/turbo
# feat: task table widget

### Description

Implementation of the "task table" component of our new UI. The table is comprised of the task names and the task status which displays an ASCII representation of when a task started and finished.

### Testing Instructions

Added some unit testing for stateful parts of task table and the task duration rendering. I held off on the unit testing of the task table rendering as I expect us to change styling a bit.

Added example binary that fakes some task execution, this if meant to take place of unit test for rendering. Please play around with it!

https://github.com/vercel/turbo/assets/4131117/03ced974-d676-425e-b7a4-edacfab2a4a2



Closes TURBO-2419
